allennlp/modules/highway.py

```diff
@@ -31,18 +30,26 @@ def __init__(self,
                  activation: Callable[[torch.Tensor], torch.Tensor] = torch.nn.functional.relu) -> None:
         super(Highway, self).__init__()
         self._input_dim = input_dim
-        self._layers = torch.nn.ModuleList([torch.nn.Linear(input_dim, input_dim * 3)
+        self._layers = torch.nn.ModuleList([torch.nn.Linear(input_dim, input_dim * 2)
                                             for _ in range(num_layers)])
         self._activation = activation
+        for layer in self._layers:
+            # We should bias the highway layer to just carry its input forward.  We do that by
+            # setting the bias on `B(x)` to be positive, because that means `g` will be biased to
+            # be high, to we will carry the input forward.  The bias on `B(x)` is the second half
+            # of the bias vector in each Linear layer.
+            layer.bias[input_dim:].data.fill_(1)
```

Comment:
Won't this be overridden by later initialization and this should rather be part of that?

Reply:
This is the default initialization.  By default now, we do no initialization, and just use whatever's in the code.  So, yes, if you want to change this, you can, but it will be initialized this way by default.

Reply:
I see, didn't realize all the default init was gone now.

Reply:
Yeah, it caused a lot of problems.  My first run of BiDAF had awful results because of a bug in the default initialization; we discovered a couple of hours ago that Mark's runs of the SRL model had accidentally re-initialized the embedding layer...  Better to just leave default initialization to the code, because pytorch initializes its modules in a mostly-reasonable way, anyway.